packages/patternfly-4/react-core/src/components/Tooltip/styles.js

```diff
@@ -0,0 +1,71 @@
+import { StyleSheet } from '@patternfly/react-styles';
+
+// inject the base tippy styles ('tippy.js/dist/tippy.css')
+export const tippyStyles = StyleSheet.parse(
```

Comment:
This can just use `injectGlobal` it will be faster since parse runs some regexs on the string.